src/lib.rs

```diff
@@ -1,8 +1,9 @@
 use anyhow::{Context, Result};
 use camino::Utf8PathBuf;
 use std::fmt;
-use std::fs::File;
-use std::io::{BufRead, BufReader};
+
+use tokio::fs::File;
+use tokio::io::{AsyncBufReadExt, BufReader};
```

Comment:
This replacement almost looks the same, but with `tokio::` instead of `std::`. Only difference is that `tokio` doesn't have `BufRead`, with another trait in place (due to interactions with `std`) and the not-yet-stable `stream` feature (which will be "async iterators" in the future)